src/basic_recipes/basic_recipes.jl

```diff
@@ -413,7 +413,10 @@ function plot!(plot::Annotations)
                 (text, startpos), color, tsize, alignment, rotation, justification, lineheight
             c = to_color(color)
             rot = to_rotation(rotation)
-            pos = layout_text(text, startpos, tsize, f, alignment, rot, model, justification, lineheight)
+            pos = layout_text(text, tsize, f, alignment, rot, model, justification, lineheight)
+
+            pos = [p .+ to_ndim(Point3f0, startpos, 0) for p in pos]
```

Comment:
Why not just 
```suggestion
            pos = pos .+ to_ndim(Point3f0, startpos, 0)
```
?

Reply:
because you are not supposed to add points without broadcasting, it throws a staticarrays deprecation warning